### ISSUE TITLE
The tag `owasp-crs` does not exist on security.stackexchange.com hence nobody can ask questions about it

### ISSUE THREAD

**Issue by renepupil** [coreruleset/coreruleset#3774]
I think the title is self-explanatory, link: https://security.stackexchange.com/questions/tagged/owasp-crs

Users need 300 points to create new tags. Why link to a help page that CAN NOT help?

**Comment by fzipi** [comment]
Can you express yourself in a gentler way? We are friendly. 😄

**Comment by fzipi** [comment]
Thanks. I was referring specifically to:
> "Who had the brilliance of linking to a help page that CAN NOT help?"

**Comment by fzipi** [comment]
Will remove my comments so we can start again.

**Comment by renepupil** [comment]
I rather focus on the actual issue, others might also be blocked by it.

**Comment by fzipi** [comment]
We are already focusing on it. But in the meantime, there are other ways to contact people, e.g. Slack.

**Comment by renepupil** [comment]
@fzipi I understand, but "Help and support" on "new issue" ONLY links to SO.

I'll ask an SO user if he can add it...

**Comment by fzipi** [comment]
Can you point me where is that? Is it the website?

**Comment by fzipi** [comment]
I've already asked the broader owasp community here: https://owasp.slack.com/archives/C04T40NND/p1721653372051619

**Comment by renepupil** [comment]
@fzipi With Slack using my workmail, I get *It looks like there isn’t an account on OWASP tied to this email address.*

**Comment by fzipi** [comment]
See https://coreruleset.org/20181003/owasp-crs-slack/

**Comment by fzipi** [comment]
> > Can you point me where is that? Is it the website?
> 
> https://github.com/coreruleset/coreruleset/issues/new/choose => And then click "Help and Support" button
> 
> @fzipi Does this help?

Yes, now I see this is in https://github.com/coreruleset/coreruleset/blob/main/.github/ISSUE_TEMPLATE/config.yml.

Weird that nobody found this out in the previous 5 years. Let's change it!

**Comment by avidouglen** [comment]
I have created the `owasp-crs` tag on Security.SE - https://security.stackexchange.com/questions/tagged/owasp-crs (btw I am a moderator there). 
And on Stackoverflow https://stackoverflow.com/questions/tagged/owasp-crs

**Comment by renepupil** [comment]
@avidouglen Awesome, thank you very much.

@fzipi Can I close this, or do you still want to change something the config yaml file?

**Comment by fzipi** [comment]
I think we are good, closing.
